pkgpanda/__init__.py

```diff
@@ -442,7 +447,14 @@ def validate_username(username):
         if not re.match(username_regex, username):
             raise ValidationError("Username must begin with `dcos_` and only have a-z and underscore after that")
 
-    def add_user(self, username):
+    @staticmethod
+    def validate_group(group):
+        try:
+            grp.getgrnam(group)
```

Comment:
http://unix.stackexchange.com/questions/11477/what-are-the-allowed-group-names-for-groupadd -- should follow the conservative rules there for what is allowed.

The goal with the `validate_username` stuff is that we find bugs early / before we launch a cluster giving people a much shorter cycle.